sympy/physics/continuum_mechanics/beam.py

```diff
@@ -1728,6 +1741,147 @@ def plot_ild_reactions(self, subs=None):
 
         return PlotGrid(len(ildplots), 1, *ildplots)
 
+    def _solve_for_ild_shear(self):
+        """
+
+        Helper function for solve_for_ild_shear(). It takes the unsubstituted
+        copy of the load equation and uses it to calculate shear force equation.
+
+        """
+        x = self.variable
+        shear_force = -integrate(self._original_load, x)
+
+        return shear_force
+
+    def solve_for_ild_shear(self, dist, val, *reactions):
+        """
+
+        Determines the Influence Line Diagram equations for shear at a
+        specified point under the effect of a moving load.
+
+        Parameters
+        ==========
+        dist : Integer
+            Distance of the specified point
```

Comment:
Elaborate this to specify that the point is where we intend to determine the shear and it is the distance from the start of the beam